operations/products.md

```diff
@@ -59,16 +59,14 @@ Returns all products offered together with the specified services.
                 "Wellness": false,
                 "CityTax": false
             },
-            "Price": {
```

Comment:
WTF? I know it was some time ago but do you recall what was the reason for this change here? @fpindej 
 
https://mews.slack.com/archives/C01TCGUJ5FV/p1678901419745069?thread_ts=1678874405.905289&cid=C01TCGUJ5FV